parameter_test.go

```diff
@@ -0,0 +1,285 @@
+package tfe
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestParametersList(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	orgTest, orgTestCleanup := createOrganization(t, client)
+	defer orgTestCleanup()
+
+	psTest, _ := createPolicySet(t, client, orgTest, nil, nil)
+
+	vTest1, _ := createParameter(t, client, psTest)
+	vTest2, _ := createParameter(t, client, psTest)
```

Comment:
I understand that this is all copy/pasted, but maybe rename all these to `pTest1`?